src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/MsQuicStream.cs

```diff
@@ -18,6 +18,7 @@ internal sealed class MsQuicStream : QuicStreamProvider
         // Delegate that wraps the static function that will be called when receiving an event.
         internal static readonly StreamCallbackDelegate s_streamDelegate = new StreamCallbackDelegate(NativeCallbackHandler);
 
+        // The state is passed to msquic and returned in msquic callback handler.
```

Comment:
nit: "returned" makes me think as it's not a parameter of the callback but a return value
```suggestion
        // The state is passed to msquic and then it's passed back by msquic to the callback handler.
```